installer/datafiles/AIX.data

```diff
@@ -48,6 +48,10 @@ fi
 #include DeleteSoftLinkToSudo
 DeleteSoftLinkToSudo
 
+%Preuninstall_850
+#include DeleteSoftLinkToTmpDir
+DeleteSoftLinkToTmpDir
```

Comment:
No real point in creating a new section here. I would just update the existing section with something like:

%Preuninstall_800
#include DeleteSoftLinkToSudo
#include DeleteSoftLinkToTmpDir

DeleteSoftLinkToSudo
DeleteSoftLinkToTmpDir

Do this in HPUX.data, Linux.data, and SunOS.data as well.